src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
```

Comment:
3. Code readability:
   - The code is concise and well-formatted, but adding comments for complex logic or explaining the intent behind certain decisions can improve readability for future maintainers.